### PR TITLE
record result totals

### PR DESCRIPTION
**Pull request by AaronJhaj**
- run result totals at end of summary and details formatter
- refactored code
- introduced constants
- unit tests
- minor updates

### CODE REVIEW

File: pkg/formatters/runsFormatter.go [hunk 1]
```diff
@@ -16,8 +17,31 @@ import (
...
+	HEADER_END_TIME       = "end-time"
+	HEADER_DURATION       = "duration(ms)"
+	HEADER_BUNDLE         = "bundle"
+	HEADER_REQUESTOR      = "requestor"
+	HEADER_RUN_LOG        = "run-log"
+	HEADER_METHOD_NAME    = "method"
+	HEADER_METHOD_TYPE    = "type"
 )
 
+var RESULT_POSSIBILES = []string{RUN_RESULT_TOTAL, RUN_RESULT_PASSED, RUN_RESULT_PASSED_WITH_DEFECTS, RUN_RESULT_FAILED, RUN_RESULT_ENVFAIL, RUN_RESULT_FAILED_WITH_DEFECTS}
```
Comment (techcobweb) [inline pkg/formatters/runsFormatter.go:43]: They aren't all possibilities. Total isn't a possibility? They are total labels ?

**Review by techcobweb — APPROVED** [review]
Approved, though I think re-naming that array might be a good idea.

File: pkg/formatters/runsFormatter.go (hunk 1 above)
Comment (AaronJhaj) [reply pkg/formatters/runsFormatter.go:43]: agreed will change

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**